data/command-monitoring/legacy/find.json

```diff
@@ -413,8 +413,9 @@
       ]
     },
     {
-      "description": "A successful find event with a getmore and the server kills the cursor",
+      "description": "A successful find event with a getmore and the server kills the cursor (<= 4.4)",
       "ignore_if_server_version_less_than": "3.1",
+      "ignore_if_server_version_greater_than": "4.4",
```

Comment:
This test appears to be failing on the PR's patch build.

The current specified behavior assumes we do track the batch size from the user supplied limit. The Go driver explicitly decided not to as part of GODRIVER-1773.

The skip description needs to be updated here: https://github.com/mongodb/mongo-go-driver/blob/69f95615de85a03ee8bfe6b661686010215b106c/mongo/integration/unified/unified_spec_runner.go#L28:L28